matrix/helper_functions.hpp

```diff
@@ -20,64 +20,39 @@ bool is_finite(Type x) {
 #endif
 }
 
+/**
+ * Wrap value to stay in range [low, high)
+ *
+ * @param x input possibly outside of the range
+ * @param low lower limit of the allowed range
+ * @param high upper limit of the allowed range
+ * @return wrapped value indside the range
+ */
+template<typename Type>
+Type wrap(Type x, Type low, Type high) {
+    Type range = high - low;
+    x = fmod(x - low, range);
+    if (x < 0)
```

Comment:
Would you mind using braces here? It helps prevent unintentional errors. Imagine someone adding another statement after `x += range;` with the same indentation.

Reply:
Sure sorry, I know the convention, I fell back to my old code style when reading through examples.
astyle enforces this on PX4...